webclient/xml_generator/definitions/amcr_230522.xsd.xml

```diff
@@ -66,22 +66,22 @@
           </xs:sequence>
         </xs:complexType>
       </xs:element>
-      <xs:element name="historie" minOccurs="0" maxOccurs="unbounded" type="amcr:historieType"/> <!-- "{historie.Historie}" -->
+      <xs:element name="historie" minOccurs="0" maxOccurs="unbounded" type="amcr:historieType"/> <!-- "{historie.historie_set.datum_zmeny}" -->
```

Comment:
@pesikj Pozor, tady mi změna přijde nesprávná. Typ je `historieType` a celý element je definován až níže v XSD. Zde je bylo jen řečeno, z čeho se má čerpat odkaz, ale půjde o komplexní typ s více podřazenými elementy.

Reply:
@motyc Jenže ten typ nemá být v komentáři, ten je dán atributem type. V komentáři je třeba uvést mapování, tj. jakým způsobem získat data. Datový typ ani názvy tříd tam být nemají, pomocí nich není možné k datům přistoupit.

Reply:
@pesikj Tak se klidně domluvme na správné syntaxi. Přes datum změny to ale určitě nepůjde. Potřebujeme nějak odkázat na všechny potomky. Upravím pak všechny výskyty i pro další obdobné případy. Já jsem se právě snažil definovat, že se má brát z pole `projekt.historie` a přes vazbu dál z modelu `Historie`.